src/lib/smart-query/Query.ts

```diff
@@ -6,14 +6,21 @@ import { UserQueryPart } from "./UserQueryPart";
 export default class SmartQuery {
     parts: UserQueryPart[] = [];
 
+    id: number = null;
     name: string = null;
     userInput: string = "";
 
+    static async loadWithUQI(queryId: string): Promise<SmartQuery> {
```

Comment:
Is UQI = user query id?

Reply:
Yes, it's to indicate that the function expect a string of the format `~usq:<id>`.
I've added a type to be clearer